sdk/src/store.rs

```diff
@@ -2075,6 +2077,46 @@ impl Store {
         }
     }
 
+    pub async fn save_to_stream_remote_signed(
```

Comment:
Is this an API we want to support?

Reply:
Not really, but I don't know how we can support adobe_c2pa without it yet.